resources/monitoring/charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 5.2.1
+version: 6.1.9
+appVersion: 7.3.3
```

Comment:
We decided to delete `appVersion` field as it is mostly forgotten to be maintained: https://github.com/kyma-project/kyma/commit/a83c6821641471141dfd547cebcec0abfbb23530. Could you please delete it from all the subcharts?

Reply:
Done